packages/opentelemetry-types/src/metrics/Metric.ts

```diff
@@ -33,9 +33,6 @@ export interface MetricOptions {
    */
   unit?: string;
 
-  /** The list of label keys for the Metric. */
-  labelKeys?: string[];
```

Comment:
This is same as https://github.com/open-telemetry/opentelemetry-js/pull/463/files#r340342807